harmonic/sensors.md

```diff
@@ -8,7 +8,28 @@ to other models in our world. We will use three different sensors:
 an IMU sensor, a Contact sensor and a Lidar sensor. We will also
 learn how to launch multiple tasks with just one file using `gz launch`.
 
-You can find the final world of this tutorial [here](https://github.com/gazebosim/docs/blob/master/harmonic/tutorials/sensors/sensor_tutorial.sdf)
+You can find the final world of this tutorial showing all these plugins in use [here.](https://github.com/gazebosim/docs/blob/master/ionic/tutorials/sensors/sensor_tutorial.sdf)
+You may also find an extensive set of world examples with many possible sensors and actuation capabilities in [`gz-sim/examples/worlds`](https://github.com/gazebosim/gz-sim/tree/gz-sim8/examples/worlds) for individual examples.
+The full set of sensors can be found in the [`gz-sensors` project](https://github.com/gazebosim/gz-sensors).
```

Comment:
Similar to Fortress, these links should be for the corresponding distro, Harmonic (gz-sim8)